src/data/contentful/teaser-linked/ctf-teaser-linked-gql.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import {DefaultQueryParameters} from "@src/data/contentful/component.typings";
+import {
+  useCtfComponentTeaserLinkedQuery
+} from '@src/data/contentful/teaser-linked/__generated/ctf-teaser-linked.generated';
+import { TeaserLinked } from '@src/mapping/contentful/teaser-linked';
+
+export function CtfTeaserLinkedGql(props: DefaultQueryParameters) {
+  const data = useCtfComponentTeaserLinkedQuery({
+    id: props.id,
+    locale: props.locale,
+    preview: props.preview,
+  });
+
+  if (data.data?.teaserLinked) {
+    return <TeaserLinked data={data} />
```

Comment:
Only data should be returned in the data layer. Please use the mapping to link data to components.

Reply:
It is linking the data to the TeaserLinked component in the mapping folder.
import { TeaserLinked } from '@src/mapping/contentful/teaser-linked';